editor/animation/animation_state_machine_editor.cpp

```diff
@@ -252,14 +252,14 @@ void AnimationNodeStateMachineEditor::_state_machine_gui_input(const Ref<InputEv
 
 		// Use midpoint distance as bias.
 		if (close_candidates.size() > 1) {
-			float best_midpoint_dist = 1e20;
+			float best_midpoint_dist_squared = 1e20;
```

Comment:
For all the cases like these I left the default value as is, in most cases the square of the actual value isn't representable (`1e40` is smaller than the smallest value representable by `float`)

Reply:
Generally the point of these values is to be "effectively infinity" so that all other values are below it, so larger values should generally be better. I wonder if there's any reason to not use infinity itself?

Reply:
I'll separate out these cases